tutorials/monitoring/setup/index.md

```diff
@@ -72,9 +70,9 @@ In order to configure an environment for monitoring with the Particular Service
 
 ### Install ServiceControl
 
-Your monitoring environment will contain a ServiceControl instance _and_ and Monitoring instance. Both of these instance types. The main ServiceControl installer includes a desktop utility called the ServiceControl Management Utility which can be used to create and manage both types of instance.
+The main ServiceControl installer includes a desktop utility called the ServiceControl Management Utility which can be used to create and manage both types of instance.
```

Comment:
should we have names written: the `ServiceControl Management Utility`?

Reply:
I don't know that we do that for ServiceInsight, ServicePulse, or ServiceControl anywhere